pages/disclaimers/index.tsx

```diff
@@ -0,0 +1,32 @@
+/*
+ *
+ * Copyright (c) 2022 The Ontario Institute for Cancer Research. All rights reserved
```

Comment:
If we're creating new files, might well use the year that is done :)
```suggestion
 * Copyright (c) 2024 The Ontario Institute for Cancer Research. All rights reserved
```